packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -347,6 +364,22 @@ class ScriptTransformer {
     const inputCode = typeof input === 'string' ? input : input.code;
     const inputMap = typeof input === 'string' ? null : input.map;
 
+    // Use cached babel-plugin-istanbul or fallback to synchronous require
+    let babelPluginIstanbul;
```

Comment:
Why do we still have this block after using `requireOrImportModule`? Should we just throw error here without using `require` anymore?
```
if (!this._babelPluginIstanbul === null) {
    throw new Error(
          'babel-plugin-istanbul is required when using coverageProvider: "babel". ' +
            'Install it with `npm install --save-dev babel-plugin-istanbul` or ' +
            'use coverageProvider: "v8" if you want to disable babel instrumentation.',
        );
}
```

- The line 397 should be `this._babelPluginIstanbul`